src/TwoDays.tsx

```diff
@@ -179,7 +183,8 @@ function ActionisedMessage({
     "author-f",
   ]);
 
-  const isAction = messageDoc.content.startsWith("/me ");
+  const isAuthorAction = messageDoc.content.startsWith("/me ");
```

Comment:
I wonder whether we should just check for `/me` (without a space) so that you can do `/me's hands are cold`

Reply:
Done!